messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/DiscoveryGroupAdd.java

```diff
@@ -62,6 +65,25 @@ private DiscoveryGroupAdd() {
     }
 
     @Override
+    protected void populateModel(OperationContext context, ModelNode operation, Resource resource) throws OperationFailedException {
+        super.populateModel(context, operation, resource);
+
+        final PathAddress address = PathAddress.pathAddress(operation.require(OP_ADDR));
+        final String name = address.getLastElement().getValue();
```

Comment:
The 2 lines should be:

String name = context.getCurrentAddressValue();

Same effect but less code and uses some cached data inside the context so a bit more efficient.